lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -361,6 +363,34 @@ def stack_operation(cfn, stack_name, operation):
             time.sleep(5)
     return {'failed': True, 'output':'Failed for unknown reasons.'}
 
+
+def get_changeset(module, stack_params, cfn):
+    """Create a change set, get it and delete it."""
+    randomchars = [random.choice(string.ascii_letters + string.digits) for x in range(0, 10)]
+    name = 'ansible-cloudformation-check-' + ''.join(randomchars)
+    try:
+        change_set = cfn.create_change_set(ChangeSetName=name, **stack_params)
+        while True:
+            description = cfn.describe_change_set(ChangeSetName=change_set['Id'])
+            if description['Status'] in ['CREATE_COMPLETE', 'FAILED']:
+                break
+            time.sleep(5)
+        cfn.delete_change_set(ChangeSetName=change_set['Id'])
+        return description
+    except Exception as err:
```

Comment:
If you look at `boto_exception` it seems fairly generic to me. It's also used inside `except Exception` in other places in this module. Also, when running this without network, I seem to get `botocore.vendored.requests.exceptions.ConnectionError` and `botocore.exceptions.EndpointConnectionError` instead of `IOError`